README.md

```diff
@@ -15,3 +15,206 @@ A JuMP extension to use parameter in constraints RHS
 [gitter-url]: https://gitter.im/JuliaOpt/StochasticDualDynamicProgramming.jl?utm_source=share-link&utm_medium=link&utm_campaign=share-link
 [gitter-img]: https://badges.gitter.im/JuliaOpt/StochasticDualDynamicProgramming.jl.svg
 [discourse-url]: https://discourse.julialang.org/c/domain/opt
+
+## Welcome to ParameterJuMP
+
+ParameterJuMP adds new methods created on top of JuMP to use constant
+parameters in optimization problems.
+
+To enable the usage of ParameterJuMP the optimization model must
+be constructed with the function:
+
+`ModelWithParam(args...)`
```

Comment:
Why not do
```julia
...
```
It looks better and have color syntaxing

Reply:
Agreed!